app/helpers/urls.js

```diff
@@ -14,34 +14,35 @@ const getUrl = (urlObject, url, request, secBtn, currentUrl) => {
   const { dependentQuestionYarKey, dependentAnswerKeysArray, urlOptions, nonDependentAnswerKeysArray = [] } = urlObject
   const { thenUrl, elseUrl, nonDependentUrl } = urlOptions
 
-  const dependentAnswer = getYarValue(request, dependentQuestionYarKey)
-
-  // handle unique input key (can be generalised later)
-  if (dependentQuestionYarKey === 'SolarPVCost') {
```

Comment:
your PR deleted my changes :P 

Atm, the suggested change would still cause UAT defect 15 issue, but this if block would fix that

This is also why those 2 tests were failing